src/iceberg/executor/penguins.py

```diff
@@ -0,0 +1,177 @@
+"""
+Author: Aymen Alsaadi
+License: MIT
+Copyright: 2019-2020
+"""
+
+from __future__ import print_function
+import os
+import csv
+import radical.entk as re
+
+from .executor import Executor
+from ..discovery import Discovery
+
+class Penguins(Executor):
+    '''
+    :Class Penguins:
+    This class instantiates the Penguins use case.
+    :Additional Parameters:
+    :input_path: The path to the input images
+        :ouptut_path: Path to the output images
+        :model: The model name 
+        :model_path: Path of a custom model
+        :epoch: number of epochs (300) 
+    '''
+    # pylint: disable=too-many-arguments
+    def __init__(self, name, resources, project=None, input_path=None,
+                 output_path=None, gpu_ids=None, model=None,
+                 model_path=None, epoch=None):
+
+        super(Penguins, self).__init__(name=name,
+                                    resource=resources['resource'],
+                                    queue=resources['queue'],
+                                    walltime=resources['walltime'],
+                                    cpus=resources['cpus'],
+                                    gpus=resources['gpus'],
+                                    project=project)
+        self._gpu_ids = gpu_ids
+        self._data_input_path = input_path
+        self._output_path = output_path
+        self._model_name = model
+        self._model_path = model_path
+        self._epoch = epoch
+        self._req_modules = None
+        self._pre_execs = None
+        self._env_var = os.environ.get('VE_PENGUINS')
+        if self._res_dict['resource'] == 'xsede.bridges':
```

Comment:
I would suggest you use a `try except` block here or introduce an `else` to account for a resource other than Bridges.